x-pack/plugin/rollup/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/rollup/50_date_nanos.yml

```diff
@@ -0,0 +1,167 @@
+setup:
+  - skip:
+      version: " - 8.6.99"
```

Comment:
SHould this be 8.7.99...we are on 8.8.0 now. After backporting it we will change the version in the other pR right?

Reply:
I see you disabled the test using AwaitsFix.